src/couch/src/couch_users_db.erl

```diff
@@ -75,7 +76,7 @@ save_doc(#doc{body={Body}} = Doc) ->
         Body3 = proplists:delete(?PASSWORD, Body2),
         Doc#doc{body={Body3}};
     {ClearPassword, "pbkdf2"} ->
-        Iterations = list_to_integer(config:get("couch_httpd_auth", "iterations", "1000")),
+        Iterations = list_to_integer(config:get("couch_httpd_auth", "iterations", "10000")),
```

Comment:
You can't change the default here without good reason; there are likely reasons it is set to 1000 not 10000. It is also an unrelated change to the proposed addition of `bcrypt`, isn't it? This is the `pbkdf2` section.

Any change to a `config:get` block must be matched with parallel changes to `rel/overlay/etc/default.ini`.

Reply:
That's right. I simply noticed that [the default value in the `authenticate` function in couch_httpd_auth.erl is 10000](https://github.com/apache/couchdb/blob/master/src/couch/src/couch_httpd_auth.erl#L401) so I thought this was a misspelling.
Wasn't aware of `rel/overlay/etc/default.ini` but I guess I should just roll back this change, should I?